.travis.yml

```diff
@@ -99,3 +116,4 @@ after_failure:
   - cd $TRAVIS_BUILD_DIR
   - cat config.log
   - cat -n ./tests/boost-test-output*
+ 
```

Comment:
Nothing should be changed here

Reply:
Rolled back in https://github.com/precice/precice/pull/421/commits/7c2ec4b1f06467d37244bf51177e0eccb134d453